### ISSUE TITLE
vignette does not illustrate reset_dsc

### ISSUE THREAD

**Issue by stephens999** [stephens999/dscr#47]
need to have some discussion of features like this somewhere. perhaps in vignette.

**Comment by ramanshah** [comment]
Agreed - one way to do this might be to make a "mistake" with one of the `one_sample_location` methods and then blow it away. We can mention in the text how different subsets of scenarios and methods can be reset, or how the entire dsc can be blown away as well.

Do you think this will remain digestible, or should we just do a separate, even more basic kind of `dscr` vignette to illustrate the reset verbs?
